build-scripts/sign_rpms.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -o xtrace
+
+TAG=${1}
+KEY=${2}
+BASE_TAG=`echo "${TAG}" | rev | cut -d '-' -f2- | rev`
+echo ${BASE_TAG}
+
+signdir=$(mktemp -d)
```

Comment:
No... sign_unsigned.py calls `os.rmdir()` on the working directory we created after clearing it out.